util/util.go

```diff
@@ -58,3 +63,50 @@ func AddImageNames(store storage.Store, image *storage.Image, addNames []string)
 	}
 	return nil
 }
+
+func ContextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemContext, error) {
+	ctx := &types.SystemContext{
+		RegistriesDirPath: c.GlobalString("registries.d"),
+		DockerCertPath:    c.String(flagPrefix + "cert-dir"),
+		// DEPRECATED: keep this here for backward compatibility, but override
+		// them if per subcommand flags are provided (see below).
+		DockerInsecureSkipTLSVerify: !c.GlobalBoolT("tls-verify"),
+		OSTreeTmpDirPath:            c.String(flagPrefix + "ostree-tmp-dir"),
```

Comment:
We'll also want to set the `SignaturePolicyPath` field here, if one's been passed on the command line.